.circleci/config.yml

```diff
@@ -49,7 +52,7 @@ jobs:
           condition:
             and:
               - equal: [main, << pipeline.git.branch >>]
-              - equal: ["21.1.0", << parameters.node-version >>]
+              - equal: ["22", << parameters.node-version >>]
```

Comment:
22 hits LTS in October, so this is probably safe. Either way, we shouldn't use 21 since it's not supported anymore.